example/App.tsx

```diff
@@ -335,13 +354,20 @@ function OpenLastExampleSetting() {
 interface MainScreenItemProps {
   name: string;
   onPressItem: (name: string) => void;
+  enabled: boolean;
 }
 
-function MainScreenItem({ name, onPressItem }: MainScreenItemProps) {
+function MainScreenItem({ name, onPressItem, enabled }: MainScreenItemProps) {
   return (
-    <RectButton style={[styles.button]} onPress={() => onPressItem(name)}>
+    <RectButton
+      enabled={enabled}
+      style={[
+        styles.button,
+        !enabled && { backgroundColor: 'rgb(220, 220, 220)', opacity: 0.3 },
```

Comment:
Could this also be moved to `styles` instead of being inline?

Reply:
Done n [b4a01e3](https://github.com/software-mansion/react-native-gesture-handler/pull/3135/commits/b4a01e3f0dc4d00411205338d0945b8b8cc95957)